JavaScript/JavaScript-Classes.md

```diff
@@ -0,0 +1,171 @@
+#JavaScript Functional Classes
+
+###Introduction
+
+JavaScript is a prototypal object-oriented language, meaning that objects can inherit directly from other objects.  This is a bit different to those who have learned to develop in other langauges like Java or C# where a subclass will inherit from a superclass, and objects are instances of classes. 
```

Comment:
@alwinsol, you say 'objects can inherit directly from other objects'. Do you mean 'objects can inherit properties directly from other objects'? It would be good to be more specific here about what is inherited.